### PR TITLE
[hevcd] Enable the SPS/PPS error reporting

### PR DESCRIPTION
**Pull request by UngTengEn**
Issue:N/A
Test:sample_decode with mfxExtDecodeErrorReport

Signed-off-by: Ung, Teng En <teng.en.ung@intel.com>

### CODE REVIEW

File: _studio/mfx_lib/decode/h265/src/mfx_h265_dec_decode.cpp [hunk 1]
```diff
@@ -1084,6 +1084,16 @@ mfxStatus VideoDECODEH265::DecodeFrameCheck(mfxBitstream *bs, mfxFrameSurface1 *
 
         MFXMediaDataAdapter src(bs);
 
+#if (MFX_VERSION >= 1025)
+        mfxExtBuffer* extbuf = (bs) ? GetExtendedBuffer(bs->ExtParam, bs->NumExtParam, MFX_EXTBUFF_DECODE_ERROR_REPORT) : NULL;
+
+        if (extbuf)
+        {
+            ((mfxExtDecodeErrorReport *)extbuf)->ErrorTypes = 0;
```
Comment (dmitry-gurulev) [inline _studio/mfx_lib/decode/h265/src/mfx_h265_dec_decode.cpp:1092]: A C++ style `reinterpret_cast` is used below but here the naked C one. Please align.

Comment (UngTengEn) [reply _studio/mfx_lib/decode/h265/src/mfx_h265_dec_decode.cpp:1092]: Hi, seems there is also some are using naked C as well.  Is this are required?

Comment (UngTengEn) [reply _studio/mfx_lib/decode/h265/src/mfx_h265_dec_decode.cpp]: Add another patch to align the code style.

**Review by dmitryermilov — APPROVED** [review]

**Comment by dmitryermilov** [comment]
@wangyan-intel , please review.

**Review by wangyan-intel — APPROVED** [review]
LGTM

**Pull request merged**